apps/api/v2/src/modules/webhooks/services/webhooks.service.ts

```diff
@@ -0,0 +1,56 @@
+import { CreateWebhookInputDto } from "@/modules/webhooks/inputs/create-webhook.input";
+import { WebhooksRepository } from "@/modules/webhooks/webhooks.repository";
+import { ConflictException, Injectable, NotFoundException } from "@nestjs/common";
+import { Webhook, Prisma } from "@prisma/client";
+
+@Injectable()
+export class WebhooksService {
+  constructor(private readonly webhooksRepository: WebhooksRepository) {}
+
+  async createUserWebhook(userId: number, body: CreateWebhookInputDto): Promise<Webhook> {
+    const existingWebhook = await this.webhooksRepository.getUserWebhookByUrl(userId, body.subscriberUrl);
+    if (existingWebhook) {
+      throw new ConflictException("Webhook with this subscriber url already exists for this user");
+    }
+    return this.webhooksRepository.createUserWebhook(userId, body);
+  }
+
+  async createEventTypeWebhook(eventTypeId: number, body: CreateWebhookInputDto): Promise<Webhook> {
```

Comment:
Shall we split `WebhooksService` into `UserWebhooksService` and `EventTypesWebhooksService` to not have this service grow too big and since user webhook creation and event types webhook creation are called by different controllers.